firmware/arduino/chessboard/circleFunction.ino

```diff
@@ -32,13 +35,16 @@ enum Quarters{
 // Makes a full circle of the given size (0=largest, NUM_CIRCLES-1=smallest) 
 // starting from the given quadrant (0=top, 1=left, 2=bottom, 3=right).
 // calculatePulsesPerSlope() must be called before makeCircle()
-void makeCircle(int circle, int firstQuarter)
+void makeCircle(int circle)
 {
     // firstPass is used to make the loop run for the first quadrant
     // despite the fact that quarter will be equal to first quarter.
     bool firstPass = true;
     int slopeIndex;
 
+    // Circle 0 starts at the top, circle 1 at the left, circle 2 at the bottom, continuing in a counterclockwise fashion
+    int firstQuarter = circle % 4;
```

Comment:
I assume the % 4 is in case we have more than 4 circles. Can you add that as a comment?